processing/src/test/java/org/apache/druid/frame/channel/ReadableByteChunksFrameChannelTest.java

```diff
@@ -327,9 +328,11 @@ public void testWriteReadInterleaved() throws IOException
           }
 
           iteration++;
+          totalSize += chunk.length;
 
           // Write next chunk.
           final ListenableFuture<?> addVal = channel.addChunk(chunk);
+          Assert.assertEquals(channel.getBytesAdded(), totalSize);
```

Comment:
Should we add the bytes size check across all the UT's ?

Reply:
Sounds fine to me. Added them.